index.bs

```diff
@@ -2043,9 +2101,16 @@ attributes.
         specified [[#attachment]].
 
     :   <dfn>requireResidentKey</dfn>
-    ::  This member describes the [=[RP]=]'s requirements regarding [=resident credentials=].
-        If the parameter is set to [TRUE], the authenticator MUST create a
-        [=client-side-resident public key credential source=] when creating a [=public key credential=].
+    ::  This member is retained for backwards compatibility but is deprecated in favour of providing *residentKey*.
+        It will be ignored if *residentKey* is supplied and understood by the user-agent. When provided, it describes the 
+        [=[RP]=]'s requirements regarding [=resident credentials=]. If the parameter is set to [TRUE], the authenticator MUST create 
+        a [=client-side-resident public key credential source=] when creating a [=public key credential=].
```

Comment:
Throughout this paragraph and the next:
- `*requireResidentKey*` => `{{requireResidentKey}}`
- `*residentKey*` => `{{residentKey}}`

Reply:
Agreed - done